docs/adr/0001-resource-identifiers.md

```diff
@@ -1,6 +1,6 @@
 # Resource Identifiers
 
-* Status: proposed
+* Status: superseded by [0003-uri-variables](0003-uri-variables.md)]
```

Comment:
```suggestion
* Status: superseded by [0003-uri-variables](0003-uri-variables.md)
```

Reply:
thx fixed in https://github.com/api-platform/core/pull/4441